pkg/util/kube/cluster.go

```diff
@@ -0,0 +1,26 @@
+package kube
+
+import (
+	"context"
+
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/cluster-api/controllers/remote"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewClusterClients(ctx context.Context, c client.Client, cluster client.ObjectKey) (*kubernetes.Clientset, dynamic.Interface, error) {
+	restConfig, err := remote.RESTConfig(ctx, c, cluster)
+	if err != nil {
+		return nil, nil, err
+	}
+	kubeClient, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return nil, nil, err
+	}
+	dynamicClient, err := dynamic.NewForConfig(restConfig)
+	if err != nil {
+		return nil, nil, err
```

Comment:
Would it be a bad behavior to return the kubeClient?
```suggestion
		return kubeClient, nil, err
```

Reply:
Not sure, but I guess the idiomatic go way would be in the case of an error to return nil for all other return params. I don't think I've stumbled upon a case where this was not the case :thinking:.

Reply:
good point :)